help/mathjax.md

```diff
@@ -59,20 +57,20 @@ link to set a cookie:
     <a href="javascript:setMathjaxCookie()" id="mathjax_toggle">Disable MathJax</a>
 
 When MathJax is disabled, mathematical expressions are displayed as text
-with the original $\\TeX$ delimiters.
+with the original <span class="mathjax">$\\TeX$</span> delimiters.
 
 Note that on any particular MathJax-enabled page in arXiv, not all
 expressions may be renderable. So turning MathJax on or off may have no
 effect on the visual appearance of certain math expressions.
 
 ### As an arXiv submitter, how can I make the best use of MathJax?
 
-MathJax supports a very large subset of $\\TeX$ mathematics, including
+MathJax supports a very large subset of <span class="mathjax">$\\TeX$</span> mathematics, including
 AMS math and symbols. arXiv's implementation of MathJax does not support
-user-defined macros. For example, if an abstract contains $\\rtwo$ as a
-shortcut for $\\mathbf{R}\^2$, then MathJax will not be able to render
-this expression. Submitters should use only standard $\\LaTeX$ and
-AMS-$\\LaTeX$ macros in paper titles and abstracts.
+user-defined macros. For example, if an abstract contains <span class="mathjax">$\\rtwo$</span> as a
+shortcut for <span class="mathjax">$\\mathbf{R}\^2$</span>, then MathJax will not be able to render
+this expression. Submitters should use only standard <span class="mathjax">La$\\TeX$</span> and
+AMS-La<span class="mathjax">$\\TeX$</span> macros in paper titles and abstracts.
 
 Also, since '\<' is a reserved character in HTML, special care must be
 taken in its use. Usually just adding a space after it will suffice for

```

Comment:
In viewing on beta, the entity reference that I think is supposed to display as `&lt;` is coming up as just `<` (line 78)

Reply:
Fixed